Verificaiton.md

```diff
@@ -0,0 +1,38 @@
+# License Verification Platform Technical Documentation
+The purpose of this document is to provide documentation for concepts relating to the [License Verification Platform](https://verificaiton.clicknclear.com) that do not fall under the API documentation.
+
+## Signup Fields and Signup Keys
+When setting up a season or event you may want to collect custom data during signup.
+
+To collect this data we recommend using Signup fields, which can be found on the season edit / event edit pages.
+When setting up a Signup field, you will notice the input 'Field Key'.
+
+Field keys can be used to identify certain fields between any season/event that you may be running.
+
+Our API endpoints will return the entries along with any signup field answers along with its associated field key.
```

Comment:
NIT: double usage of "along" may read better as:

> Our API endpoints will return the entries along with any signup field answers and the associated field keys (if set).

Etc.

Reply:
ba64c5a